gcode_metadata/metadata.py

```diff
@@ -618,6 +618,21 @@ def get_metadata(path: str, save_cache=True, filename=None):
     return metadata
 
 
+def biggest_resolution(thumbnails: dict):
+    """Get the thumbnail with the biggest resolution from the list of
+    thumbnails"""
+    max_width = max_height = 0
+
+    for resolution in thumbnails:
+        width, height = map(int, resolution.split('x'))
+        ratio = width / height
+
+        # Check if the current resolution has a larger key pair and
+        # meets the ratio condition
+        if 1 <= ratio <= 2 and width * height > max_width * max_height:
```

Comment:
to moc nechápu - max_width * max_height bude tedy vždy nula?  to bude podmínka splněna vždy ne?

Reply:
Špatná verze :))